.devcontainer/Dockerfile

```diff
@@ -0,0 +1,37 @@
+FROM mcr.microsoft.com/devcontainers/base
+
+RUN apt-get update && export DEBIAN_FRONTEND=noninteractive \
+  && apt-get -y install --no-install-recommends \
+  curl \
+  yamllint \
+  graphviz \
+  && rm -rf /var/lib/apt/lists/*
+
+# renovate: datasource=github-releases depName=moby/buildkit
+ENV BUILDKIT_VERSION=v0.21.0
+ENV BUILDKIT_URL=https://github.com/moby/buildkit/releases/download/${BUILDKIT_VERSION}/buildkit-${BUILDKIT_VERSION}.linux-amd64.tar.gz
```

Comment:
is it possible to introduce checksum verification to verify all these downloaded binaries ?